bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -1643,4 +1643,24 @@ private static boolean areEntriesOfSegmentStoredInTheBookie(LedgerMetadata ledge
         }
         return firstStoredEntryId != LedgerHandle.INVALID_ENTRY_ID;
     }
+
+    /**
+     * returns boolean value specifying if the ensemble of the segment is
+     * adhering to the ensemble placement policy for the given writeQuorumSize
+     * and ackQuorumSize.
+     *
+     * @param ensembleList
+     *            ensemble of the segment
+     * @param writeQuorumSize
+     *            writeQuorumSize of the ledger
+     * @param ackQuorumSize
+     *            ackQuorumSize of the ledger
+     * @return <tt>true</tt> if the ledger is adhering to
+     *         EnsemblePlacementPolicy
+     */
+    public boolean isEnsembleAdheringToPlacementPolicy(List<BookieSocketAddress> ensembleList, int writeQuorumSize,
+            int ackQuorumSize) {
+        return bkc.getPlacementPolicy().isEnsembleAdheringToPlacementPolicy(ensembleList, writeQuorumSize,
+                ackQuorumSize);
```

Comment:
I might be missing something; I don't see where this isEnsembleAdheringToPlacementPolicy implemented

Reply:
it is implemented in each EnsemblePlacementPolicy

Reply:
Is that not part of this pull request?